runtime_scan/pkg/scanner/job_managment.go

```diff
@@ -284,19 +284,25 @@ func (s *Scanner) createJob(data *scanData) (*batchv1.Job, error) {
 		removeCISDockerBenchmarkScannerFromJob(job)
 	}
 	job.SetName(jobName)
-	job.SetNamespace(podContext.namespace)
+	if job.Namespace == "" {
+		// Set job namespace to pod namespace if not set in the template.
+		// May be overridden later if support for image pull secrets or SA credentials is needed.
+		job.SetNamespace(podContext.namespace)
```

Comment:
@FrimIdan  do you know a reason why we couldn't default to the release namespace here? and then only use the pod namespace for the imagePullSecret case?

Reply:
Just because we want to keep the old behavior.

Reply:
👍 sounds good